cmake-modules/AzureDoxygen.cmake

```diff
@@ -58,6 +58,7 @@ function(generate_documentation PROJECT_NAME PROJECT_VERSION)
         set(DOXYGEN_REPEAT_BRIEF NO)
 
         doxygen_add_docs(${PROJECT_NAME}-docs
+            ./inc ./README.md
```

Comment:
Will this have any impact on the "root readmes" being published for each package? For example, if we exclude https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/storage/azure-storage-blobs/README.md does that mean, the published docs won't have:
https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-storage-blobs/12.12.0/index.html

Reply:
It shouldn't - it adds the readme.md file from each project in the documentation.